### PR TITLE
Add detection of RKWard running interactive session.

### PR DESCRIPTION
**Pull request by devillemereuil**
Tested and working on RKWard 0.7.0b.

The principle is that RKWard is loading its own package named "rkward" when it starts, so if the package is loaded, it means that R has been loaded by RKWard.

Let me know if I misunderstood how the code works.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -39,9 +39,19 @@ r_app_stdx <- function(stream) {
...
+rkward_stdx <- function(stream) {
+  interactive() &&
+    is_rkward() &&
+    (is_stdout(stream) || is_stderr(stream))
+}
+
 is_supported <- function(stream) {
   is_option_enabled() &&
-    (isatty(stream) || r_studio_stdx(stream) || r_app_stdx(stream))
+    (isatty(stream) || r_studio_stdx(stream) || r_app_stdx(stream)) || rkward_stdx(stream)
```
Comment (gaborcsardi) [inline R/utils.R]: Are you sure you want to close that paren there?

Comment (devillemereuil) [reply R/utils.R]: I am sure that I don't want to, sorry. The new commit should fix this. By the way, is there a reason for not simply using:
```R      
(is_stdout(stream) || is_stderr(stream)) && interactive()
```
?

**Comment by gaborcsardi** [comment]
Thanks!

File: R/utils.R (hunk 1 above)
Comment (gaborcsardi) [reply R/utils.R]: Probably could do. Just being cautious, in case some environments might not support `\r`.

**Pull request merged**